controllers/admin.js

```diff
@@ -75,9 +78,9 @@ exports.postEditProduct = (req, res, next) => {
   const updatedPrice = req.body.price;
   Product.findById(prodId)
     .then((product) => {
-      if (product.userId.toString() !== req.user._id.toString()) {
-        return res.redirect("/");
-      }
+      // if (product.userId.toString() !== req.user._id.toString()) {
```

Comment:
make sure that you delete all products and admin user creates them all again so this can work properly